griptape/tasks/tool_task.py

```diff
@@ -84,7 +84,11 @@ def run(self) -> BaseArtifact:
             subtask.after_run()
 
             if isinstance(subtask.output, ListArtifact):
-                self.output = subtask.output[0]
+                first_artifact = subtask.output[0]
+                if isinstance(first_artifact, BaseArtifact):
+                    self.output = first_artifact
+                else:
+                    self.output = ErrorArtifact(f"Output is not an Artifact: {type(subtask.output[0])}")
```

Comment:
should this throw an error or is this a case we want to catch

Reply:
Good point, maybe this is worth raising.